MariaDB/server
# MDEV-28232: There is missing breaks for 10.7 in MariaDB version 10.8

- [x] *The Jira issue number for this PR is: MDEV-28232*

## Description
There is several missing Breaks and conflict for MariaDB version 10.7 in Debian `control`  file which can be lead to situation where upgrading does not work

## How can this PR be tested?
Salsa-CI build can be found on: https://salsa.debian.org/illuusio/mariadb-server/-/pipelines/367970

```
debian/autobake-deb.sh
cd ..
apt-get install mariadb-server
dpkg -i *.deb
```
There should not be any problems with breaks and conflicts

## Basing the PR against the correct MariaDB version

## Backward compatibility
Should not break anything
